themes/base.theme.bash

```diff
@@ -433,8 +433,8 @@ function rbfu_version_prompt {
 }
 
 function chruby_version_prompt {
-	if declare -f -F chruby &> /dev/null; then
-		if declare -f -F chruby_auto &> /dev/null; then
+	if _command_exists chruby; then
+		if _command_exists chruby_auto; then
```

Comment:
Since this was specifically testing for being a function, you might want to use [`_is_function`](https://github.com/Bash-it/bash-it/blob/master/lib/helpers.bash#L167)